app/uk/gov/hmrc/cgtpropertydisposals/service/onboarding/AuditService.scala

```diff
@@ -70,38 +73,64 @@ class AuditServiceImpl @Inject()(
     httpStatus: Int,
     body: String,
     path: String
-  )(implicit hc: HeaderCarrier, executionContext: ExecutionContext): Unit = {
-
-    val detail = SubscriptionResponseEvent(
-      httpStatus,
-      body
-    )
-
-    sendEvent(
-      "subscriptionResponse",
-      detail,
-      hc.toAuditTags("subscription-response", path)
-    )
-
-  }
+  )(implicit hc: HeaderCarrier, executionContext: ExecutionContext): Unit =
+    Try {
+      Json.parse(body)
+    } match {
+      case Failure(exception) => {
+        logger.warn(s"{Could not send subscription response audit message: $exception}")
```

Comment:
doesn't look like what this log message is saying is true since we're sending an event below. We don't really care what went wrong with trying to read the body as JSON - it'll become apparent in the splunk event anyways

Reply:
log message taken out